packages/calcite-components/src/components/rating/rating.e2e.ts

```diff
@@ -519,8 +521,15 @@ describe("calcite-rating", () => {
       await labels[2].click();
       await page.waitForChanges();
 
+      const hoveredEl = await page.findAll("calcite-rating >>> .star.hovered");
+      const selectedEl = await page.findAll("calcite-rating >>> .star.selected");
+      const focusedElId = labels[2].getAttribute("for");
+
       expect(await element.getProperty("value")).toBe(0);
       expect(changeEvent).toHaveReceivedEventTimes(1);
+      expect(hoveredEl.length).toBe(3);
+      expect(await isElementFocused(page, `[for=${focusedElId}]`, { shadowed: true })).toBe(true);
```

Comment:
Not critical, but is it possible to query using the `star` class via `label` or `label` attributes? This would make the test less brittle in case the implementation changes.